src/utils.ts

```diff
@@ -1198,3 +1198,24 @@ export function markdownToWebviewHtml(
   cleanHtml = cleanHtml.replace(/&lt;/g, "<").replace(/&gt;/g, ">");
   return cleanHtml;
 }
+
+/**
+ * Shows an error notification with a button that opens a link when clicked.
+ * @param {string} errorMessage - The error message to display.
+ * @param {string} [buttonLabel="Read Documentation"] - The label for the button (default: "Read Documentation")
+ * @param {string} linkUrl - The URL to open when the button is clicked.
+ * @returns {Promise<void>} - A promise that resolves when the notification is shown.
+ */
+export async function showErrorNotificationWithLink(
```

Comment:
This function could live closer to the notification logic like `src/logger` and extended for other use cases.